packages/storage/src/__tests__/index.test.ts

```diff
@@ -489,6 +489,21 @@ describe('Storage Utils', () => {
         });
       });
 
+      describe('clear', () => {
+        it('should remove all items', async () => {
+          const item1 = storage.defineItem(`${storageArea}:one`);
+          const item2 = storage.defineItem(`${storageArea}:two`);
+          await fakeBrowser.storage[storageArea].set({
+            one: 1,
+            two: 2,
+          });
+
+          await storage.clear(storageArea);
+          expect(await item1.getValue()).toBeNull();
+          expect(await item2.getValue()).toBeNull();
```

Comment:
We can simplify this test a little, it doesn't need to rely on the storage item implementation. We can check if storage was cleared using just `fakeBrowser`.


```suggestion
          await fakeBrowser.storage[storageArea].set({
            one: 1,
            two: 2,
          });

          await storage.clear(storageArea);
          expect(await fakeBrowser.storage[storageArea].get()).toEqual({});
```